src/sys/DISK.ASM

```diff
@@ -257,7 +257,7 @@ load_file:                                      ; Load file starting at cluster
 
 
 ; WRITE FILE TO DISK: Writes contents of file buffer to a file on the disk and applies any filesystem changes
-; IN: [filename] = Valid filename (run conv_x3_83 to set this up); [dir_id] = Directory ID; CX = File size
+; IN: [filename] = Valid filename; [dir_id] = Directory ID; CX = File size; [file_buffer_segment]:[file_buffer_pointer] = File buffer
```

Comment:
File buffer is now variable to allow for writing directories without overwriting I/O buffer.